code/modules/power/solar.dm

```diff
@@ -52,6 +54,9 @@ TYPEINFO(/obj/machinery/power/tracker)
 		id = "aisat"
 	New()
 		..()
+		if (isnull(src.targetstar))
+			SPAWN(2 SECONDS)
+				src.targetstar = global.starlist[1]
```

Comment:
Is this needed? `sun.calc_position` should update this every 2.3 seconds.  This seems like it would likely often cause the tracker to run for one cycle on the wrong star?

Reply:
i honestly don't remember typing this out. I don't think it's needed at all, lol. Just gonna remove it.